### DISCUSSION TITLE
Precise Line Highlighting

### DISCUSSION THREAD

**Discussion by LamprosPitsillos** [xournalpp/xournalpp#4498] [Q&A]
Is there a way to highlight text by lines ? 
What i mean is that i cant perfectly highlight the text from a line precisely.
I just need to select the text and "change the background" to my selected color

**Comment by rolandlo** [comment]
For text from the PDF background there is the PDF toolbox (in the nightly version) to highlight text. For text inserted via the text tool there is no such option unfortunately.

**Comment by Adnan-Bacic** [comment]
im sorry, but am i the only one getting weird sexual junk spam mail from this thread for some reason?

**Comment by rolandlo** [comment]
No, you're not. I also got some a couple of hours ago. Looks like it has been deleted automatically.

**Comment by Adnan-Bacic** [comment]
turns out a lot of other people are also getting emails from github to random repos they never interacted with:
https://old.reddit.com/r/ProgrammerHumor/comments/zy4ct8/getting_mentioned_in_a_random_issue_on_a_random/
